pkgs/os-specific/linux/uhk-udev-rules/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, uhk-agent, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "uhk-udev-rules";
+  version = lib.getVersion uhk-agent;
+  inherit (uhk-agent);
+  dontBuild = true;
+  dontUnpack = true;
+  installPhase = ''
+     install -D -m 644 ${uhk-agent.out}/rules/50-uhk60.rules $out/lib/udev/rules.d/50-uhk60.rules
+  '';
+
+  meta = {
+    description = ''
+    udev rules for UHK keyboards
+    from https://ultimatehackingkeyboard.com.
+    Users must be in the input group. E.g. \n
+    users.extraGroups.input.members = [ "userName" ];\n
+    '';
+    inherit (uhk-agent.meta) license;
+    maintainers = [ lib.maintainers.lourkeur ];
```

Comment:
```suggestion
    maintainers = [ lib.maintainers.ngiger ];
```

Reply:
Thanks for spotting this error. I would like to apologize for not thinking enough about copy/pasting code!

Reply:
Apology accepted! I wasn't sure how to react at first but I figured it was probably a copy/paste. It's no big deal.